### DISCUSSION TITLE
Load LoRA into Kandisnky2.2 prior and decoder

### DISCUSSION THREAD

**Discussion by SimonJonsson1999** [huggingface/diffusers#7478] [Q&A]
I have trained A LoRA on Kandinsky 2.2 prior and decoder, but i don't know how to load these weights into the model. As far as i understand it, other models use the pipeline.load_lora_weights function but this does not exist. 
Help is appreciated!

**Comment by tolgacangoz** [comment]
Hi @SimonJonsson1999,
Could you try to inherit from `LoraLoaderMixin`?
[KandinskyV22Pipeline](https://github.com/huggingface/diffusers/blob/544710ef0fe83c3ea87744c21a550d46e088a3c4/src/diffusers/pipelines/kandinsky2_2/pipeline_kandinsky2_2.py#L64)
```diff
-class KandinskyV22Pipeline(DiffusionPipeline):
+class KandinskyV22Pipeline(DiffusionPipeline, LoraLoaderMixin):
```
[KandinskyV22PriorPipeline](https://github.com/huggingface/diffusers/blob/544710ef0fe83c3ea87744c21a550d46e088a3c4/src/diffusers/pipelines/kandinsky2_2/pipeline_kandinsky2_2_prior.py#L84)
```diff
-class KandinskyV22PriorPipeline(DiffusionPipeline):
+class KandinskyV22PriorPipeline(DiffusionPipeline, LoraLoaderMixin):
```
